exampleSite/content/docs/comments-support.md

```diff
@@ -57,6 +57,22 @@ disqusShortname = ""
 
 And that's all!
 
+### Isso
+
+Isso is a lightweight alternative to Disqus. You need to have a Isso server running somewhere, then set up the following options in your `config.toml` file:
```

Comment:
Might be useful to mention comment.sh here in the docs as well as in the PR?

Reply:
Not sure this is the right place to endorse them. I added it in the PR as a convenience if someone wants to test it, but that's not the same thing as linking to them in the docs.